census/metadata.csv

```diff
@@ -0,0 +1,3 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
+census.syncs.rows_processed,count,,item,,The number of rows processed by a single sync.,0,census,Rows Processed,
+census.syncs.sync_completed,count,,unit,,Number of syncs completed,0,census,Sync Completion,
```

Comment:
```suggestion
census.syncs.sync_completed,count,,unit,,Number of syncs completed.,0,census,Sync Completion,
```

Reply:
Feel free to update the other metric description instead if you'd rather not use punctuation in the metric table (up to you)